presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -754,7 +753,7 @@ private void testCreatePartitionedBucketedTableAs(HiveStorageFormat storageForma
                 // make sure that we will get one file per bucket regardless of writer count configured
                 getParallelWriteSession(),
                 createTable,
-                "SELECT count(*) from orders");
+                "SELECT COUNT(*) FROM orders");
```

Comment:
All thse `count` -> `COUNT` / `from` -> `FROM` are unrelated changes. I would suggest to do it in a separate PR :)

Reply:
Thanks for the review. Yes, this is likely IntelliJ corrected it automatically during changes. Let me correct it.